mods/tools/hoes.lua

```diff
@@ -20,6 +20,7 @@ local function Hoe_Use(itemstack, user, pointed_thing)
 			end
 			
 			local soilness = minetest.get_item_group(node.name, "soil")
+			local loamness = minetest.get_item_group(node.name, "loam")
 			if soilness ~= 0 then
```

Comment:
Can we compress these two if .. elseif blocks into 1 to reduce code repetition?
Something like:
```lua
if (soilness ~= 0 or loamness ~= 0) then
    local param2 = ...
    ...
    minetest.swap_node(pos, {
        name = qts.select(soilness ~= 0, "overworld:dirt_tilled", "overworld:loam_tilled"), 
        param2 = param2
    })
    ...
else
...
```